ngraph_creator/operations/src/Conv_2d.cpp

```diff
@@ -203,11 +210,28 @@ std::shared_ptr<ngraph::Node> Conv_2d::createNode() {
         }
     }
 
-    auto inputNode = getInputNode<float>(0);
-    auto filterNode = getInputNode<float>(1);
+    std::shared_ptr<ngraph::Node> inputNode, filterNode, biasNode;
+
+    const auto& inputIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 0);
+    const auto& filterIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 1);
+    const auto& biasIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 2);
+
+    if (checkInputOperandType(0, (int32_t)OperandType::TENSOR_FLOAT32)) {
+        inputNode = getInputNode<float>(0);
+        filterNode = getInputNode<float>(1);
+        biasNode = getInputNode<float>(2);
+    } else if (checkInputOperandType(0, (int32_t)OperandType::TENSOR_QUANT8_ASYMM)) {
+        inputNode = getInputNode<uint8_t>(0);
+        filterNode = getInputNode<uint8_t>(1);
+        biasNode = getInputNode<int>(2);
+
+        inputNode = DequantizeNode(inputNode, inputIndex, ngraph::element::f32);
+        filterNode = DequantizeNode(filterNode, filterIndex, ngraph::element::f32);
+        biasNode = DequantizeNode(biasNode, biasIndex, ngraph::element::f32);
```

Comment:
The NNAPI Documentation mentions bias :
zeroPoint of 0 and bias_scale == input_scale * filter_scale

But, this would pick the scale and zeropoint from the biasIndex operand right ?